pyproject.toml

```diff
@@ -40,7 +40,7 @@ standard_library = ["typing_extensions"]
 
 [tool.ruff]
 target-version = "py38"
-line-length = 120
+line-length = 88
```

Comment:
Need to select the I001 to enable RUFF isort: https://docs.astral.sh/ruff/rules/unsorted-imports/